### DISCUSSION TITLE
Cant remove the `Don't get locked out of your account.` notification?  `Download your recovery codes or add a passkey`

### DISCUSSION THREAD

**Discussion by RedCMD** [community/community#78705] [Other Feature Feedback, Questions, & Ideas] [answered]
## Bug

I've downloaded the codes and used ctrl+shift+r, but it won't go away
![image](https://github.com/community/community/assets/33529441/34df321f-7feb-42cd-9855-4f1483e0beb9)
> Don't get locked out of your account. [Download your recovery codes](https://github.com/settings/auth/recovery-codes?two_factor_holiday_warning=true) or [add a passkey](https://github.com/settings/security?two_factor_holiday_warning=true#passkeys) so you don't lose access when you get a new device.

also clicking the [x] is very slow, why does the webpage need to request the server to close it?

**Comment by jhwz** [comment]
I also can't remove it, I've downloaded the codes and added a passkey... annoying!

**Reply by tookser** [reply]
Also as far as I remember, I have 2FA and one times codes list, so it is very confusing.

**Comment by mkll** [comment]
Very annoying considering I've already followed all the recommended security steps.

**Comment by RedCMD** [comment, marked as answer]
Well.
It suddenly just disappeared for me
same for everyone else?

**Reply by mkll** [reply]
Same thing. Disappeared.

**Comment by mesopa** [comment]
I also have the 2FA and backup codes, started showing again, really annoying. Probably another `sleep` bug going on.
